frontend/src/index.css

```diff
@@ -89,12 +83,34 @@ code {
 }
 
 .logo-div {
-  padding: 5px;
+  padding: 8px;
   display: inline-block;
   position: relative;
+  top: -7%;
   width: 192px !important;
   height: 192px !important;
   border-radius: 50%;
-  background: #fff;
-  margin: 20px 0;
+  margin: 17px 0;
+}
+
+/* about page css *********************************/
+
+.lowermid {
```

Comment:
All of this should be moved into a `components/static/index.css` file and imported from there.